src/components/pages/Weather/Weather.js

```diff
@@ -14,22 +24,81 @@ class Weather extends React.Component {
         this.setState({ weather });
       })
       .catch(err => console.error('error with getting the weather', err));
+
+    weatherRequests.getIsCurrent(this.props.uid)
+      .then((currentLocal) => {
+      // if isCurrent is true
+        if (currentLocal.isCurrent === true) {
```

Comment:
I'm getting an error here because I don't have a currentLocal defined (because none of my weather locations have isCurrent as true. You should be able to account for that by checking instead for if you have a currentLocal defined. Maybe something like `if (currentLocal)`

Reply:
fixed and re pushed to test